packages/babel-preset-fbjs/plugins/__tests__/__snapshots__/inline-requires-test.js.snap

```diff
@@ -0,0 +1,112 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`inline-requires deletes \`Foo\` if unused: deletes \`Foo\` if unused 1`] = `
+"
+const Foo = require('Foo');
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+
+"
+`;
+
+exports[`inline-requires does not inline \`Bar\` (ignored): does not inline \`Bar\` (ignored) 1`] = `
+"
+const Bar = require('Bar'); Bar;
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+const Bar = require('Bar');Bar;
+"
+`;
+
+exports[`inline-requires does not inline properties on \`Bar\` (ignored): does not inline properties on \`Bar\` (ignored) 1`] = `
+"
+const Baz = require('Bar').Baz; Baz;
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+const Baz = require('Bar').Baz;Baz;
+"
+`;
+
+exports[`inline-requires ignores \`Foo\` when not assigned to a variable: ignores \`Foo\` when not assigned to a variable 1`] = `
+"
+const Foo = require('Foo'); Foo; Foo;
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+require('Foo');require('Foo');
+"
+`;
+
+exports[`inline-requires inlines \`Foo\` before it is declared: inlines \`Foo\` before it is declared 1`] = `
+"
+const doSomething = () => Foo; const Foo = require('Foo');
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+const doSomething = () => require('Foo');
+"
+`;
+
+exports[`inline-requires inlines \`Foo\` in multi-variable declarations: inlines \`Foo\` in multi-variable declarations 1`] = `
+"
+const Foo = require('Foo'), Bar = 123, Baz = 456; Foo;
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+const Bar = 123,
+      Baz = 456;require('Foo');
+"
+`;
+
+exports[`inline-requires inlines \`Foo\` multiple times: inlines \`Foo\` multiple times 1`] = `
+"
+const Foo = require('Foo'); Foo; Foo;
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+require('Foo');require('Foo');
+"
+`;
+
+exports[`inline-requires inlines \`Foo\` properties with destructuring: inlines \`Foo\` properties with destructuring 1`] = `
+"
+const {Baz} = require('Foo'); Baz;
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+const { Baz } = require('Foo');Baz;
```

Comment:
This is incorrect.

Reply:
Note: that's per se _not_ incorrect, as we rely on the destructuring Babel transform to lower `const {a: c} = require('b')` to `const c = require('b').a`; but that's however not what the existing test asserted.